addons/core/translations/resources/en-us.yaml

```
@@ -583,6 +590,23 @@ role:
       descendants:
         title: All Descendants
         sub-title: Add all of the orgs and the projects in the orgs that are underneath this scope
+    actions:
+      manage-scopes: Manage Scopes
+      add-custom-scopes:
+        text: Add custom scopes
+        help: Customize which scopes are added to this role.
+    form:
+      this:
+        label: Add this scope
+        help: Add this scope.
+      children:
+        label: Add all children
+        help:
+          0: Add all of the orgs underneath this scope.
+          1: Add all of the projects underneath the org.
```

Should it say

```suggestion
          1: Add all of the projects underneath this org.
```
Since the previous one says `this scope`?

---

I took the text directly from the design so i didn't notice the difference. Good catch!